ochazuke/__init__.py

```diff
@@ -118,8 +118,13 @@ def needsdiagnosis_from_db():
                     count=item.count)
                 timeline.append(hourly_count)
             timeline = json.dumps(timeline)
```

Comment:
@laghee 

This is where it goes wrong.

```python
>>> timeline = []
# timeline is a list
>>> hourly_count = {'timestamp': '34878', 'count': 123}
>>> timeline.append(hourly_count)
>>> timeline
[{'timestamp': '34878', 'count': 123}]
# timeline is a list of dictionaries
>>> import json
>>> type(json.dumps(timeline))
<class 'str'>
>>> resp = {'timeline': json.dumps(timeline)}
>>> resp
{'timeline': '[{"timestamp": "34878", "count": 123}]'}
# resp is a dictionary containing a string for timeline, not a JSON object.
```

So two issues with the code. 

1. Adding a string to the timeline key.
2. Returning a dictionary instead of a JSON.

Probably better to keep working with dictionaries and list as long as we are in the realm of python, then convert to JSON when it's time to send the response.